airbyte-integrations/connectors/source-netsuite/source_netsuite/streams.py

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC
+from collections import OrderedDict
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional
+
+# Basic full refresh stream
+import requests
+import zeep
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams.http import HttpStream
+from netsuitesdk import NetSuiteConnection
+from netsuitesdk.api.base import ApiBase
+
+
+class NetsuiteSoapStream(HttpStream, ApiBase, ABC):
+    def __init__(self, nc: NetSuiteConnection, name: str, config: dict, concurrency_limit: int = 1):
+        self.config = config
+        super().__init__()
```

Comment:
what's the point of doing this? we are narrowing the interface by this

Reply:
Seems like, indeed, we should not initiate the super class again.